beyond-essence/inventory/toggle-switches.md

```diff
@@ -89,16 +75,46 @@ To decide when to use a toggle switch, here are some clues:
+As toggle switches typically allow merchants to enable or disable features or modes that have a significant effect on their workflow and data might be lost when they deactivate a previously activated feature, make sure the label (and description, if applicable) are precise and make it easy for the merchant to understand the consequences of activating/deactivating the toggle.
+
+* Be short, precise and direct
+* Start the toggle label with a verb and describe what changing its default state will do, e.g. “Allow visitors to make purchases on your website.” (exception: “Website is live”)
+* If there is a static description text underneath the toggle, describe what will happen when the switch is activated.
+Start with “If this toggle is activated…”
+* If the description text only appears after the feature is toggled on, describe what will happen when the switch is turned off.
+Start with: “If you toggle off FEATURE…”
 * Don't use neutral or ambiguous phrases.
 * Avoid asking questions.
 
-```
-Track stock level
-```
+**DO**: "If this toggle is activated, there will be a cart displayed in the header of your website as well as an "Add to cart" button on each product.
+Be sure to save any changes."
+
+_Precisely explains what happens when the switch is turned on._
+
+**DON'T**: "A cart will be displayed in the header of your website."
+
+_Too ambigous, unclear if this is what happens when the toggle is turned on or off._
+
+**ALSO DON'T**: "Want to display a cart in the header of your website?"
+
+_The question format makes in unclear for the merchant if they need to activate or deactivate the toggle for the desired outcome._
+
+## Position
+
+Toggle switches may be positioned on a card together with further UI elements that relate to a specific topic.
 
 ## Design
 
-An activated on toggle switch comes with a white checkmark icon on a green background, whereas a deactivated toggle switch is visualised via a white "close icon" on a beige background.
-Immediately after toggling the switch from off to on more information and/or functionality is displayed in an area with light beige background below the toggle switch.
+A toggle switch looks like a classic on/off switch.
+The two different states give clear visual feedback so that the merchant is always aware if the toggle switch is turned on or off.
+
+* A white checkmark icon on a green background, toggled to the right, represents an active feature or mode.
+* A white cross on a beige background, toggled to the left, represents a deactivated feature or mode.
+
+On the right side of the toggle switch, a label clearly explains the action that can be taken.
+
+Depending on the complexity of the feature or mode, additional content may be shown below the toggle switch.
+
+{% image_custom image="/assets/img/pages/essence/images-for-variations.png" width="100" %}
+
+For further information on the exact visualisation of addtional content, consult the design team.
```

Comment:
add**i**tional :)

Furthermore, I'd say visuali**z**ation as the official company language is en-US :)